autogluon/training/docker/0.8/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,68 @@
+ARG PYTHON_VERSION=3.9.16
+
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.13.1-cpu-py39-ubuntu20.04-sagemaker
+
+# PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK - see https://github.com/autogluon/autogluon/issues/2534
+# ENV PYTORCH_SKIP_CUDNN_COMPATIBILITY_CHECK=1
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Install dependencies
+RUN apt-get update \
+ && apt-get -y upgrade \
+ && apt-get install -y --no-install-recommends \
+ && apt-get autoremove -y \
+ # Install tesseract-ocr to support automm document classification
+ && apt-get install tesseract-ocr -y \
+ # Install rsync to support ray distributed training
+ && apt-get install rsync -y \
```

Comment:
Do we still need this for 0.8?

Reply:
Yes, we will need it as long as we want to support distributed training with ray